packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -1,154 +1,138 @@
+    }
+
+    const changeable_fields = [...getChangeableFields()];
+
+    const initial_values = {
+        document_type: {
+            id: '',
+            text: '',
+            value: '',
+            example_format: '',
+            sample_image: '',
         },
-    } = selected_country;
-
-    React.useEffect(() => {
-        // NOTE: This is a temporary filter. Remove after backend handles this from their side
-        const document_types = Object.keys(document_data);
-        const filtered_documents = ['gh', 'ng'].includes(country_code)
-            ? document_types.filter(d => d !== 'voter_id')
-            : document_types;
-
-        setDocumentList(
-            filtered_documents.map(key => {
-                const { display_name, format } = document_data[key];
-                const { new_display_name, example_format, sample_image } = getDocumentData(country_code, key);
-                const needs_additional_document = !!document_data[key].additional;
-
-                if (needs_additional_document) {
-                    return {
-                        id: key,
-                        text: new_display_name || display_name,
-                        additional: {
-                            display_name: document_data[key].additional?.display_name,
-                            format: document_data[key].additional?.format,
-                        },
-                        value: format,
-                        sample_image,
-                        example_format,
-                    };
-                }
-                return {
-                    id: key,
-                    text: new_display_name || display_name,
-                    value: format,
-                    sample_image,
-                    example_format,
-                };
-            })
-        );
-    }, [country_code, document_data]);
-
-    const onKeyUp = (e, document_name, values, setFieldValue) => {
-        const { example_format } =
-            document_name === 'document_number' ? values.document_type : values.document_type.additional;
-        const current_input = example_format.includes('-')
-            ? formatInput(example_format, current_input || e.target.value, '-')
-            : e.target.value;
-        setFieldValue(document_name, current_input, true);
-        validateFields(values);
+        document_number: '',
+        ...form_initial_values,
     };
 
-    const resetDocumentItemSelected = setFieldValue => {
-        setFieldValue(
-            'document_type',
-            {
-                id: '',
-                text: '',
-                value: '',
-                example_format: '',
-                sample_image: '',
-            },
-            true
-        );
-        setDocumentImage('');
+    const getExampleFormat = example_format => {
+        return example_format ? localize('Example: ') + example_format : '';
     };
```

Comment:
@yauheni-deriv, @likhith-deriv,   This is defined across multiple files, wouldn't it be better to move it to a shared file, where it can be imported from to avoid duplication? 
`
    const getExampleFormat = example_format => {
        return example_format ? localize('Example: ') + example_format : '';
    };
`

Reply:
Will move it as a shared function